package-audit.js

```diff
@@ -1,7 +1,7 @@
 const exec = require('child_process').exec;
 
-const AUDIT_EXCEPTIONS = ['underscore'];
-
+const AUDIT_EXCEPTIONS = ['underscore','dns-packet'];
+// dns-packet added to permit builds on 5-27-21
 const execute = (command, callback) => {
```

Comment:
just add `dns-packet` to resolutions in package.json `"dns-packet": "5.2.2"` should work